src/Compilers/Server/VBCSCompilerTests/DesktopBuildClientTests.cs

```diff
@@ -227,45 +193,12 @@ public void FallbackToCsc()
                     return 0;
                 });
 
-                var exitCode = client.RunCompilation(new[] { "/shared" }, _buildPaths).ExitCode;
+                var exitCode = client.RunCompilation(new[] { "/shared" }, _buildPaths, pipeName: _pipeName).ExitCode;
                 Assert.Equal(0, exitCode);
                 Assert.True(ranLocal);
                 Assert.Equal(1, _failedCreatedServerCount);
                 Assert.Equal(0, _serverDataList.Count);
             }
-
-            [Fact]
-            [WorkItem(7866, "https://github.com/dotnet/roslyn/issues/7866")]
-            public void RunServerCompilationThrows()
```

Comment:
This test was essentially validating the internal exception handling model of a type. This forced the type to have extra complexity and after discussing decided it was not worth the trade off. There are other tests which validate the overall scenario: handling a bad response from the server.